content/blog/sprint-review-2209.md

```diff
@@ -0,0 +1,41 @@
+Title: Wazo Platform 22.09 Released
+Date: 2022-06-21T21:08:00
+Author: The Wazo Authors
+Category: Wazo Platform
+Tags: wazo-platform, development
+Slug: release-review-2209
+Status: published
+
+Hello Wazo Platform community!
+
+Here is a short review of the Wazo Platform 22.09 release.
+
+## Bug Fixes
+- **Provisioning**: Changing the provisioning port via the API did not change the provisioning server configuration.
+- **Webhooks**: The webhooks are now tenant-aware: a webhook will only be triggered by events happening in the webhook's tenant.
+
+## Technical Features
+- **Security**: Use HTTPS by default for connections to services APIs
```

Comment:
```suggestion
```
That's not true, the HTTPS is only for swagger documentation, not API services
I'll remove this line